tasks/size_contracts.js

```diff
@@ -25,6 +19,16 @@ task(
 
   const config = hre.config.contractSizer;
 
+  const SIZE_LIMIT = 24576;
+  const UNITS = config.units == 'b' ? 'B' : config.units == 'kb' ? 'Kb' : 'KiB';
```

Comment:
"B" should be capitalized.  "k" should not.  It seems that it's conventional in computer science to capitalize the "k", but this is disallowed by SI.

```suggestion
  const UNITS = config.units == 'b' ? 'B' : config.units == 'kb' ? 'kB' : 'KiB';
```

Reply:
The `UNITS` variable can probably be removed in favor of just using `config.units`.  See comment in `index.d.ts`.

Reply:
Having made the type case-insensitive, I would leave the `UNIT` variable so that it can be formatted correctly in console.